website/docs/typescript-support.md

```diff
@@ -42,6 +42,7 @@ It is **not possible** to use a TypeScript config file in Docusaurus, unless you
 
 We recommend using [JSDoc type annotations](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html):
 
+<!-- prettier-ignore-start -->
```

Comment:
why needed?

Reply:
I had to prettier-ignore all the docusaurus config files with JSDoc because this is what Prettier turns it into, with the new line-breaking algorithm:

<table>
<thead>
<th>Before</th>
<th>After</th>
</thead>
<tr>
<td>

```js
/** @type {import('@docusaurus/types').DocusaurusConfig} */
(module.exports = {
  title: 'Docusaurus',
  tagline: 'Build optimized websites quickly, focus on your content',
  organizationName: 'facebook',
});
```

</td>
<td>

```js
/** @type {import('@docusaurus/types').DocusaurusConfig} */
(
  module.exports = {
    title: 'Docusaurus',
    tagline: 'Build optimized websites quickly, focus on your content',
    organizationName: 'facebook',
  }
);
```

</td>
</tr>

I don't particularly like that, but I also feel like this is just dodging the issue